packages/mobile/src/send/Send.tsx

```diff
@@ -168,50 +174,66 @@ class Send extends React.Component<Props, State> {
   }
 
   componentDidUpdate(prevProps: Props) {
-    const { recentRecipients, allRecipients } = this.props
+    const { recentRecipients, phoneRecipients, valoraRecipients } = this.props
 
+    // TODO: also check valoraRecipients
```

Comment:
TODO mentions `valoraRecipients`, but code seems to take it into account already.